lib/syslogger/creators.rb

```diff
@@ -4,15 +4,15 @@
 module SysLogger
   module Creators
     def self.unix_dgram_socket(socket_path)
-      proc {
+      Proc.new {
```

Comment:
I don't really care. The source I looked at (<http://batsov.com/articles/2014/02/04/the-elements-of-style-in-ruby-number-12-proc-vs-proc-dot-new/>) says to prefer `proc` over `Proc.new`, but I only googled it for 3 seconds and don't know if that's the prevailing Ruby style elsewhere.